ci/steps.yml

```diff
@@ -139,7 +165,7 @@ steps:
           python -m aqt tool --outputdir $(Build.BinariesDirectory)/Qt $(HOST) tools_mingw 8.1.0-1-202004170606 qt.tools.win32_mingw810
           [Environment]::SetEnvironmentVariable("Path", ";$(Build.BinariesDirectory)\Qt\Tools\mingw810_32\bin" + $env:Path, "Machine")
         }
-        $env:Path = "$(Build.BinariesDirectory)\Qt\Tools\$(ARCHDIR)\bin;$(Build.BinariesDirectory)\Qt\$(QT_VERSION)\$(ARCHDIR)\bin;" + $env:Path
+        $env:Path = "$(Build.BinariesDirectory)\Qt\Tools\$(ARCHDIR)\bin;$(WIN_QT_BINDIR);" + $env:Path
```

Comment:
Similar to `QT_BINDIR`, I made a `WIN_QT_BINDIR` to account for the different path separator on Windows. I'm not sure how well Powershell handles paths with forward slashes.

We could get away with not using `WIN_QT_BINDIR` here if we're not adding these tests to any of the Powershell builds. I'm adding this as a precaution in case someone wants to add such a test in the future. Whoever does that will need to either conditionally replace `$(Build.BinariesDirectory)\Qt\Tools\$(ARCHDIR)\bin` with `$(OUTPUT_DIR)\Tools\$(ARCHDIR)\bin`, or add something like a `WIN_QT_TOOLS_BINDIR` variable.